File: tes.js
const prompt=require("prompt-sync")({sigint:true}); 

function segitigaKebalik(rows) {
  for (let i = rows; i > 0; i--) {
    let spasi = " ".repeat(rows - i);
    let bintang = "*".repeat(2 * i - 1);
    console.log(spasi + bintang);
  }
}

let rows = parseInt(prompt("Masukkan jumlah baris: "));
segitigaKebalik(rows);

File: tes.py
def segitiga_kebalik(bintang):
    if bintang % 2 == 0:
        bintang += 1
    rows = (bintang + 1) // 2
    for i in range(rows, 0, -1):
        print(" " * (rows - i), end="")
        print("*" * (i * 2 - 1))

bintang = int(input("Masukkan jumlah bintang: "))
segitiga_kebalik(bintang)